.travis.yml

```diff
@@ -76,19 +68,6 @@ matrix:
       compiler: "gcc-5-release-i686"
       env: TARGET_ARCH='i686' CCOMPILER='gcc-5' CXXCOMPILER='g++-5' BUILD_TYPE='Release'
 
-    - os: linux
```

Comment:
Ticket the ARM upgrade here https://github.com/Project-OSRM/osrm-backend/issues/2699